Makefile

```diff
@@ -246,5 +246,17 @@ socketcan:
 update_codegen:
 	@python make/git_fetch.py -folder=libraries/codegen-tooling -user=uw-midsun -repo=codegen-tooling-msxiv -tag=latest -file=codegen-tooling-out.zip
 
+.PHONY: test_python
+test_python:
+	@python3 -m unittest discover -t $(PROJ_DIR)/$(PROJECT)/scripts -s $(PROJ_DIR)/$(PROJECT)/scripts -p "test_*$(TEST).py"
+
+#find projects -name "test_*.py" -print0 | xargs -0 python -m unittest
+.PHONY: test_all_python
+test_all_python:
+	for i in $$(find projects -name "test_*.py"); \
+	do											\
+		python -m unittest discover -t $$(dirname $$i) -s $$(dirname $$i) -p $$(basename $$i);							\
+	done										
```

Comment:
You need an `@` before the `for` so that it doesn't print the command. Also the backslashes are in super inconsistent positions, please either have all of them one space after the command or align them or something.